### PR TITLE
A small clean up

### PR DESCRIPTION
**Pull request by nurmukhametov**
This PS deletes a few functions that are not used in the project. It also deletes most of `#if 0` code (either debug or commented out since the beginning).

It also updates upload actions that left unupdated since https://github.com/ispc/ispc/pull/2820

### CODE REVIEW

**Review by dbabokin — CHANGES_REQUESTED** [review]

**Review by turinevgeny — APPROVED** [review]

File: src/sym.cpp [hunk 1]
```diff
@@ -316,10 +316,6 @@ std::vector<std::string> SymbolTable::ClosestVariableOrFunctionMatch(const char
 
 std::vector<std::string> SymbolTable::ClosestTypeMatch(const char *str) const { return closestTypeMatch(str, true); }
 
-std::vector<std::string> SymbolTable::ClosestEnumTypeMatch(const char *str) const {
-    return closestTypeMatch(str, false);
```
[the remark this answers is not in the document]
Comment (nurmukhametov) [reply src/sym.cpp:320]: Yes, indeed. Fixed and test was added.

[the remark this answers is not in the document]
Comment (dbabokin) [reply src/sym.cpp:320]: Probably it's better to just make it work for non-enums :) But it's up to you. If you'd like to proceed with current fix, please remove mentioning of `structsVsEnums` in the comments and you are good to go.

[the remark this answers is not in the document]
Comment (nurmukhametov) [reply src/sym.cpp:320]: Hm, I don't understand which example should trigger the corresponding warnings for structs. I tried to create something but I did only manage to get several crashes. See the issue https://github.com/ispc/ispc/issues/2844 . So, I think it is better to fix separately.

**Review by dbabokin — APPROVED** [review]
LGTM

**Pull request merged**